libs/auth-api-lib/seeders/data/scope-signature-collection.ts

```diff
@@ -0,0 +1,43 @@
+import { compose, createScope } from './helpers'
+
+export const up = compose(
+  createScope({
+    name: '@admin.island.is/signature-collection:process',
+    displayName: 'Meðmælakerfi Admin',
+    description: 'Umsjón með söfnunum meðmæla og úrvinnsla',
+    addToResource: '@admin.island.is',
+    addToClients: ['@admin.island.is/web'],
+    accessControlled: true,
+    delegation: {
+      custom: true,
+      procuringHolders: false,
```

Comment:
Both the admin scopes needs to be granted for procuration holders to work for the admin porta.